pkg/tests/ossm/smoke_test.go

```diff
@@ -65,7 +65,9 @@ func TestSmoke(t *testing.T) {
 
 			t.LogStepf("Create SMCP %s and verify it becomes ready", fromVersion)
 			assertSMCPDeploysAndIsReady(t, fromVersion)
-			assertRoutesExist(t)
+			if env.GetSMCPVersion().GreaterThan(version.SMCP_2_2) {
```

Comment:
When we run on OSSM 2.5, this will have to be updated otherwise the test will fail. It would be nice not to hardcode the version but always compute it dynamically from the highest supported version.